permute/tests/test_ksample.py

```diff
@@ -15,6 +15,10 @@ def test_worms_ksample():
     res = k_sample(worms.x, worms.y, stat='one-way anova', reps=1000, seed=1234)
     np.testing.assert_array_less(0.006, res[0])
     np.testing.assert_array_less(res[0], 0.02)
+    res2 = k_sample(worms.x, worms.y, stat=one_way_anova, reps=1000, keep_dist=True, seed=1234)
```

Comment:
For testing, I added prints for value of `res `from line 15 (printed as "Res1:") , and `res2 `(printed as "Res2:") from line 18.  Values show up as following:


```
Res1: (0.01098901098901099, 17.58271428571427)

Res2: (0.01098901098901099, 17.58271428571427, array([9.28809524e+00, 3.30785714e+00, 1.67714286e-01, 7.86523810e-01,
       1.88938095e+00, 1.10714286e-01, 2.46271429e+00, 5.63095238e-01,
       5.70752381e+00, 1.53693810e+01, 1.91438095e+00, 7.90309524e+00,
       6.19509524e+00, 4.54752381e+00, 4.09342857e+00, 3.04938095e+00,
       1.65152381e+00, 1.15303810e+01, 1.31738095e+00, 7.42752381e+00,
       4.84538095e+00, 6.41071429e+00, 6.76771429e+00, 4.37800000e+00,
       1.02115238e+01, 2.26942857e+00, 1.53510952e+01, 1.22423810e+01,
          :
          :
```

So assert added in:
line 19: length is 1000 of array being printed in res2
line 20: res and res2 are same.
line 21: see values of array printed in res2.